guides/source/layouts_and_rendering.md

```diff
@@ -103,32 +103,6 @@ In most cases, the `ActionController::Base#render` method does the heavy lifting
 
 TIP: If you want to see the exact results of a call to `render` without needing to inspect it in a browser, you can call `render_to_string`. This method takes exactly the same options as `render`, but it returns a string instead of sending a response back to the browser.
 
-#### Rendering Nothing
```

Comment:
Why did we removed completely this section from the guide? 

Should we replace it with the new `head :ok` syntax, or `body: nil` ?
(BTW which one should we use? Since this section has been removed from the guide, we don't know which one to use.)